examples/dreambooth/train_dreambooth_lora_flux.py

```diff
@@ -997,6 +1004,108 @@ def encode_prompt(
     return prompt_embeds, pooled_prompt_embeds, text_ids
 
 
+# CustomFlowMatchEulerDiscreteScheduler was taken from ostris ai-toolkit trainer:
+# https://github.com/ostris/ai-toolkit/blob/9ee1ef2a0a2a9a02b92d114a95f21312e5906e54/toolkit/samplers/custom_flowmatch_sampler.py#L95
+class CustomFlowMatchEulerDiscreteScheduler(FlowMatchEulerDiscreteScheduler):
```

Comment:
@linoytsaban I would just re-use the relevant parts from the original `CustomFlowMatchEulerDiscreteScheduler` here rather copy-pasting it entirely. For example, we don't need the `get_simas()` method 'cause we already have one inside the script. 

Also option to use this scheduler related changes should be made configurable IMO.